### PR TITLE
RD-2283 Fix cfy logs download for 3 nodes cluster

### PR DESCRIPTION
**Pull request by mabuaisha**
This Fix added in order to allow generate logs archives based on the host ip + node type instead of depending on profile manager ip. Moreover, it adds support to validate if both ssh_user & ssh_user are set before downloading logs

### CODE REVIEW

File: cloudify_cli/commands/logs.py [hunk 1]
```diff
@@ -93,6 +95,13 @@ def download(output_path, all_nodes, logger):
 
     if not output_path:
         output_path = os.getcwd()
+
+    _profile = env.profile
+    if not all([_profile.ssh_user, _profile.ssh_key]):
```
Comment (tehasdf) [inline cloudify_cli/commands/logs.py:100]: `all` for 2 items? lol maybe `and` :)

File: cloudify_cli/commands/logs.py [hunk 2]
```diff
@@ -93,6 +95,13 @@ def download(output_path, all_nodes, logger):
...
+        raise CloudifyCliError(
+            '`ssh_user` & `ssh_key` must be set before '
+            'download logs'
```
Comment (tehasdf) [inline cloudify_cli/commands/logs.py:103]: grammar plz

File: cloudify_cli/commands/logs.py [hunk 3]
```diff
@@ -93,6 +95,13 @@ def download(output_path, all_nodes, logger):
 
     if not output_path:
         output_path = os.getcwd()
+
+    _profile = env.profile
```
Comment (tehasdf) [inline cloudify_cli/commands/logs.py:99]: why set this?

File: cloudify_cli/commands/logs.py [hunk 4]
```diff
@@ -41,22 +41,23 @@ def get_host_date(conn):
...
-def _archive_logs(conn, node_type, logger):
+def _archive_logs(conn, node_type, logger, node_ip=None):
     """Creates an archive of all logs found under /var/log/cloudify plus
     journalctl.
     """
     archive_filename = 'cloudify-{node_type}-logs_{date}_{ip}.tar.gz'.format(
         node_type=node_type,
         date=get_host_date(conn),
-        ip=env.profile.manager_ip
+        ip=node_ip or env.profile.manager_ip
```
Comment (tehasdf) [inline cloudify_cli/commands/logs.py:51]: probably better to just make this always take the ip, then it won't have the env.profile dependency

**Review by tehasdf — COMMENTED** [review]
so why did this not work before?

File: cloudify_cli/commands/logs.py (hunk 4 above)
Comment (mabuaisha) [reply cloudify_cli/commands/logs.py]: Fixed

File: cloudify_cli/commands/logs.py (hunk 3 above)
Comment (mabuaisha) [reply cloudify_cli/commands/logs.py]: Updated

File: cloudify_cli/commands/logs.py (hunk 2 above)
Comment (mabuaisha) [reply cloudify_cli/commands/logs.py]: Updated

File: cloudify_cli/commands/logs.py (hunk 1 above)
Comment (mabuaisha) [reply cloudify_cli/commands/logs.py]: Done

**Review by tehasdf — APPROVED** [review]

**Pull request merged**